test/config.ts

```diff
@@ -50,7 +50,7 @@ globalSetup(async () => {
 
 const config: Config = {
   testDir: path.join(__dirname, "e2e"), // Search for tests in this directory.
-  timeout: 30000, // Each test is given 30 seconds.
+  timeout: 60000, // Each test is given 60 seconds.
```

Comment:
A couple of new the new tests are a bit longer so I increased the timeout.

Reply:
Also, since we retry tests, they may run a bit longer, hence why we need to increase the timeout.